src/ngraph/frontend/onnx_import/utils/reshape.hpp

```diff
@@ -32,23 +33,52 @@ namespace ngraph
             /// \return The new node being a 2D matrix representing flattened input node.
             std::shared_ptr<ngraph::Node> flatten(const std::shared_ptr<ngraph::Node>& node,
                                                   int axis);
-        } // namespace utils
-
-        /// \brief Permute axes according to specified axes_order parameter.
-        ///
-        /// \param node The node which axes we want to permute.
-        /// \param axes_order The permutation of node tensor axes.
-        ///
-        /// \return New node with permuted axes.
-        std::shared_ptr<ngraph::Node> reorder_axes(const std::shared_ptr<ngraph::Node>& node,
-                                                   std::vector<int> axes_order);
-
-        /// \brief Return transposed tensor (with axes in reversed order).
-        ///
-        /// \param node Input tensor we want to transpose
-        ///
-        /// \return New node with reversed dimensions.
-        std::shared_ptr<ngraph::Node> transpose(const std::shared_ptr<ngraph::Node>& node);
-    } // namespace onnx_import
 
+            /// \brief      Gets the AxisVector filled with monotonic increasing
+            ///             sequence.
+            ///
+            /// \param[in]  data_shape_size  The data shape size.
+            /// \param[in]  start_value      The start_value for sequence. Default equals 0.
+            ///
+            /// \return     The filled AxisVector.
+            ///
+            ngraph::AxisVector get_default_axis_vector(std::size_t data_shape_size,
```

Comment:
`ngraph::` is not required. You are already in `ngraph` namespace. Unless there are name clashes.

Reply:
Done.